src/app/scripts/cac/control/cac-control-directions-list.js

```diff
@@ -94,14 +94,16 @@ CAC.Control.DirectionsList = (function (_, $, MapTemplates) {
 
 
         $container.empty();
+        $container.append($html);
 
         // Show alert with link to transit agency bicycle policy for bike+transit itineraries
         if (_.includes(itinerary.modes, 'BICYCLE') && itinerary.agencies.length) {
-            var $alert = MapTemplates.bicycleWarningAlert(itinerary.agencies);
-            $container.append($alert);
+            var alert = MapTemplates.bicycleWarningAlert(itinerary.agencies);
+            var $alert = $(alert);
+            $alert.one('click', '.close', function () { $alert.remove(); });
```

Comment:
Move `.close` and `.step-by-step-header` to `options.selectors`. No hard-coded selectors!